app/assets/src/components/views/PipelineViz/PipelineViz.jsx

```diff
@@ -347,6 +347,11 @@ class PipelineViz extends React.Component {
     const updatedStagesOpened = [...this.state.stagesOpened];
     updatedStagesOpened[index] = !updatedStagesOpened[index];
     this.setState({ stagesOpened: updatedStagesOpened });
+
+    window.sessionStorage.setItem(
```

Comment:
We should probably use url for this (using `history.replaceState`)... allows sharing the state with other too. Wdyt?